ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/ovn/provision/cucushift-installer-rehearse-gcp-upi-ovn-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
```

Comment:
So this PR also include UPI OVN workflow?

Reply:
> So this PR also include UPI OVN workflow not only sdn2ovn, right?

@zhaozhanqi You are right. We do not have gcp upi ovn workflow right now.

Reply:
@zhaozhanqi @jianlinliu @lihongan Retest keep failing but have no idea about the error message. We do not have gcp-upi-ovn workflow right now. Should I create workflow: cucushift-installer-rehearse-gcp-upi-ovn first in one PR and then create workflow: cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn in another PR?

Reply:
I think if you want to make both ovn and sd2ovn in this PR, you also need to test ovn installation workflow.  for now, only sdn2ovn workflow iis tested in config file.

Reply:
I also hit the same error in another gcp-upi job and always failed at `Step installer-rehearse-gcp-upi-install-gcp`, but I don't think it is related to gcp-upi-ovn workflow. And yes we can create both workflows in one PR, you can test twice or just add the other one to 4.12 config file.
@jianlinliu Do you have any idea about the high failure rate of gcp-upi-install?

Reply:
@jianli-wei can you help check that.

Reply:
> I think if you want to make both ovn and sd2ovn in this PR, you also need to test ovn installation workflow. for now, only sdn2ovn workflow iis tested in config file.

@zhaozhanqi @jianlinliu @lihongan  Should I create below workflow: cucushift-installer-rehearse-gcp-upi-ovn under openshift-verification-tests-master__installer-rehearse-4.11.yaml in this PR too? Should workflow: cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn will work without workflow: cucushift-installer-rehearse-gcp-upi-ovn?
```
- as: installer-rehearse-gcp
  interval: 960h
  steps:
    cluster_profile: gcp-qe
    workflow: cucushift-installer-rehearse-gcp-upi-ovn
```

Reply:
> Should I create below workflow: cucushift-installer-rehearse-gcp-upi-ovn under openshift-verification-tests-master__installer-rehearse-4.11.yaml in this PR too?

If you want to add it, ideally, yes. You can run a validation job in 4.12 using the workflow.

> Should workflow: cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn will work without workflow: cucushift-installer-rehearse-gcp-upi-ovn
I think so, I did not see any dependency between them.

Reply:
https://github.com/openshift/release/pull/32107 get merged, pls sync your branch with upstream, so that upi gcp install can move on.